### PR TITLE
gnutls: build with unbound/dane support

### PR DESCRIPTION
**Pull request by schanzen**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

I know that the changes are suboptimal, especially the manual changes of CFLAGS/LDFLAGS for unbound.
gnutls does not support supplying the unbound prefix so I am not sure how to solve it.
Also, it would probably be better to make this optional?
Pointers would be helpful.

### CODE REVIEW

File: Formula/gnutls.rb [hunk 1]
```diff
@@ -31,7 +32,8 @@ def install
       --disable-heartbeat-support
       --with-p11-kit
     ]
-
+    ENV["CFLAGS"] = "-I/usr/local/Cellar/unbound/1.9.0/include"
```
Comment (javian) [inline Formula/gnutls.rb]: Are you sure these are needed ? `pkg-config` and homebrew ought to figure this out.

Comment (schanzen) [reply Formula/gnutls.rb]: Thanks for looking at it. Yes you are correct and I knew about this feature it just wasn't working.
Maybe I got it mixed up.
I added an update that removes the flags and makes the unbound dependency optional.

**Comment by schanzen** [comment]
Now I get:
gnutls:
  * Formulae should not have optional or recommended dependencies
Error: 1 problem in 1 formula detected

when I check it with brew audit. Why?

**Comment by javian** [comment]
@schanzen options are no longer allowed so if you want it in it needs to be made the default.

**Comment by schanzen** [comment]
Removed the optional. Thanks for the guidance.

**Comment by javian** [comment]
@BrewTestBot test this please

**Comment by javian** [comment]
@BrewTestBot test this please

**Comment by javian** [comment]
Thanks @schanzen for putting this together and welcome to the Homebrew community! Without people like you it wouldn't flourish and continue to grow and live as it does.

**Pull request closed without merging**